prompts.md

```
@@ -1,127 +1,254 @@
-**Prompt 1:**
+**Nota**: El proyecto se ejecutó principalmente de forma interactiva con el agente principal, no con subagentes en paralelo. La decisión de dispatchar o no subagentes en paralelo se evalúa con la skill `dispatching-parallel-agents` cuando hay 2+ tareas independientes.
 
-**Prompt 2:**
+---
 
-**Prompt 3:**
+## 3. Workflows implementados
+
+### Workflow 1: Spec-Driven Development (SDD) — principal
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)